pkg/steps/release/snapshot.go

```diff
@@ -49,12 +49,12 @@ func (r *releaseSnapshotStep) Run(ctx context.Context) error {
 }
 
 func (r *releaseSnapshotStep) run(ctx context.Context) error {
-	_, err := snapshotStream(ctx, r.client, r.config.Namespace, r.config.Name, r.jobSpec.Namespace, r.name, r.integratedStream)
+	_, err := snapshotStream(ctx, r.client, r.config.Namespace, r.config.Name, r.jobSpec.Namespace, r.name, r.integratedStream, *r.config.ReferencePolicy)
 	return err
 }
 
 // snapshotStream snapshots the source IS and the snapshot copy created
-func snapshotStream(ctx context.Context, client loggingclient.LoggingClient, sourceNamespace, sourceName string, targetNamespace func() string, targetRelease string, integratedStream *configresolver.IntegratedStream) (*imagev1.ImageStream, error) {
+func snapshotStream(ctx context.Context, client loggingclient.LoggingClient, sourceNamespace, sourceName string, targetNamespace func() string, targetRelease string, integratedStream *configresolver.IntegratedStream, refPolicy imagev1.TagReferencePolicyType) (*imagev1.ImageStream, error) {
```

Comment:
Let's pass the `refPolicy *imagev1.TagReferencePolicyType` and make a nil check to default it. Also let's see if we can properly validate the value in the config validation.